app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
```

Comment:
If you've not encountered these yet, consider taking a look at Devise's `:lockable`, `:timeoutable`, and `:confirmable` modules so you're aware of what they do.

You can find out more about them from the summaries in the bullets near the top of [Devise's README](https://github.com/plataformatec/devise), and following the links next to them to see the API docs which will give you a bit more detail.

If you want to add any or all of these modules to this app feel free, and be sure to add the associated database migrations.